theme/SubsiteRow.scss

```
@@ -0,0 +1,78 @@
+  background: transparent;
+  color: var(--rp-c-text-2, #6a6a73);
+  cursor: pointer;
+  transition:
+    background-color 0.15s ease-out,
+    border-color 0.15s ease-out,
+    color 0.15s ease-out,
+    box-shadow 0.15s ease-out;
+
+  &:hover:not(.subsite-icon--active) {
+    color: var(--rp-c-text-1, #111113);
+    background: color-mix(in srgb, currentColor 8%, transparent);
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)